php/blocks/class-loader.php

```diff
@@ -250,6 +250,22 @@ public function render_block_template( $block, $attributes ) {
 		$block_lab_attributes = $attributes;
 		$block_lab_config     = $block;
 
+		if ( ! is_admin() && ! defined( 'REST_REQUEST' ) && ! wp_doing_ajax() ) {
```

Comment:
If `REST_REQUEST` was set, but `false`, this conditional would return an unexpected result. Unfortunately, I think we need something more like:

```
if ( ! is_admin() && ( ! defined( 'REST_REQUEST' ) || ! REST_REQUEST ) && ! wp_doing_ajax() )
```

Reply:
Ah, good point.

Reply:
Sounds good, applied in https://github.com/getblocklab/block-lab/commit/76bf375addd6e81d51de6d872bb9ba86627645e2